### ISSUE TITLE
Aborted (core dumped) - tf 2.3, tf lite model converts but crashes on invoke 

### ISSUE THREAD

**Issue by albertsokol** [tensorflow/tensorflow#42923]
Hey there! 
I'm trying to convert StyleGan2 to TF lite. Currently, there seems to be an issue with invoking the synthesis blocks of the generator, so I've included a small model here which is just one synthesis block, to display the issue.

Converts fine in tf 2.3, with no issues. Then on attempting to invoke, I get the exit code
`Process finished with exit code 134 (interrupted by signal 6: SIGABRT)`
in Pycharm, and
`Aborted (core dumped)`
if running from the terminal. 

Judging from the graph in Netron, seems there are 2 Flex ops (Conv2D on 4-dim tensors in NCHW format) and everything else is built-in ops. 

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04
- TensorFlow installed from (source or binary): source - conda
- TensorFlow version (or github SHA if from source): converted with tf 2.3, tried to invoke on tf2.3 and tf-nightly 

**Command used to run the converter or code if you’re using the Python API**
Here's a colab notebook which downloads the tflite model and attempts to invoke: 
https://colab.research.google.com/drive/1g586G8gWdCrA2EQRzJ3V6PmtbnxpfQUT?usp=sharing

**Link to .tflite model**
https://drive.google.com/file/d/1yvcvTWZlgBbA6mihNJBfEiGCPVz9FqtD/view?usp=sharing

**Link to SavedModel**
https://drive.google.com/drive/folders/1JGtipSJ37p-JJSD-k4j5ZbbWgow0gHJo?usp=sharing

** Log from conversion**
[conversion_log.txt](https://github.com/tensorflow/tensorflow/files/5168414/conversion_log.txt)

```

# Here is the code used to convert the model:

converter = tf.lite.TFLiteConverter.from_keras_model(synth_const)
        converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS, tf.lite.OpsSet.SELECT_TF_OPS]
        tflite_model = converter.convert()

        with tf.io.gfile.GFile('synth_const.tflite', 'wb') as f:
            f.write(tflite_model)

```

Attempted on both the SavedModel and the tf keras functional model. 

**The output from the converter invocation**
Without GPU:
```
2020-09-03 13:00:19.623097: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
INFO: Created TensorFlow Lite delegate for select TF ops.
2020-09-03 13:00:20.461562: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-09-03 13:00:20.490338: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 3199980000 Hz
2020-09-03 13:00:20.490824: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55c893461c80 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-09-03 13:00:20.490859: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-09-03 13:00:20.495591: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2020-09-03 13:00:20.508318: E tensorflow/stream_executor/cuda/cuda_driver.cc:314] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2020-09-03 13:00:20.508384: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:169] retrieving CUDA diagnostic information for host: y4tsu-pc
2020-09-03 13:00:20.508399: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:176] hostname: y4tsu-pc
2020-09-03 13:00:20.508513: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:200] libcuda reported version is: 450.66.0
2020-09-03 13:00:20.508567: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:204] kernel reported version is: 450.66.0
2020-09-03 13:00:20.508580: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:310] kernel version seems to match DSO: 450.66.0
INFO: TfLiteFlexDelegate delegate: 2 nodes delegated out of 25 nodes with 2 partitions.

Process finished with exit code 134 (interrupted by signal 6: SIGABRT)
```
With GPU:
```
2020-09-03 13:03:30.866202: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
INFO: Created TensorFlow Lite delegate for select TF ops.
2020-09-03 13:03:31.687743: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-09-03 13:03:31.710426: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 3199980000 Hz
2020-09-03 13:03:31.711119: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x564ecd17eb30 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-09-03 13:03:31.711173: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-09-03 13:03:31.718118: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2020-09-03 13:03:31.873839: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-09-03 13:03:31.886701: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-09-03 13:03:31.887313: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x564ecd21f560 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2020-09-03 13:03:31.887326: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): GeForce GTX 1080 Ti, Compute Capability 6.1
2020-09-03 13:03:31.887331: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (1): GeForce GTX 1050 Ti, Compute Capability 6.1
2020-09-03 13:03:31.887558: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-09-03 13:03:31.887995: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:01:00.0 name: GeForce GTX 1080 Ti computeCapability: 6.1
coreClock: 1.6325GHz coreCount: 28 deviceMemorySize: 10.91GiB deviceMemoryBandwidth: 451.17GiB/s
2020-09-03 13:03:31.888041: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-09-03 13:03:31.888337: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 1 with properties: 
pciBusID: 0000:05:00.0 name: GeForce GTX 1050 Ti computeCapability: 6.1
coreClock: 1.392GHz coreCount: 6 deviceMemorySize: 3.95GiB deviceMemoryBandwidth: 104.43GiB/s
2020-09-03 13:03:31.888357: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-09-03 13:03:31.889727: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2020-09-03 13:03:31.891078: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2020-09-03 13:03:31.891344: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2020-09-03 13:03:31.892774: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2020-09-03 13:03:31.893593: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2020-09-03 13:03:31.896654: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2020-09-03 13:03:31.896761: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-09-03 13:03:31.897299: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-09-03 13:03:31.897737: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-09-03 13:03:31.898261: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-09-03 13:03:31.898550: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1843] Ignoring visible gpu device (device: 1, name: GeForce GTX 1050 Ti, pci bus id: 0000:05:00.0, compute capability: 6.1) with core count: 6. The minimum required count is 8. You can adjust this requirement with the env var TF_MIN_GPU_MULTIPROCESSOR_COUNT.
2020-09-03 13:03:31.898559: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2020-09-03 13:03:31.898586: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2020-09-03 13:03:32.301209: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-09-03 13:03:32.301234: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0 1 
2020-09-03 13:03:32.301241: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N N 
2020-09-03 13:03:32.301244: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 1:   N N 
2020-09-03 13:03:32.301444: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-09-03 13:03:32.302199: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-09-03 13:03:32.302626: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 9647 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:01:00.0, compute capability: 6.1)
INFO: TfLiteFlexDelegate delegate: 2 nodes delegated out of 25 nodes with 2 partitions.

Process finished with exit code 134 (interrupted by signal 6: SIGABRT)
```
Any idea what's going on or a possible fix for this? Should I be changing some of the ops to fit tf lite better in some way?

**Comment by abattery** [comment]
I can confirm it that this abortion is because current multiplication operator partially supports 5-D dimension inputs. Sorry for that.

**Comment by albertsokol** [comment]
Ah I see. Thanks very much for getting back to me. It seems I'll have to try and re-wire some things to make it compatible. In that case, I just have a couple of questions:
 - Are there any ops which are supported in 5D? Specifically, I'm wondering about the reshape and transpose operations. 
 - Further down in the model, I use a reshape and padding op on 6D tensors. Would this be compatible?

**Comment by albertsokol** [comment]
Have managed to figure this out without using any 4D ops. Thanks!

**Comment by yahuuu** [comment]
> I can confirm it that this abortion is because current multiplication operator partially supports 5-D dimension inputs. Sorry for that.

@abattery  Hi Abattery, I wonder when Does Transpose Operator Support >=6D?

**Comment by abattery** [comment]
Currently, there are no plans to support 6D transpose op. Could you file a separate feature request if you need it?
